### ISSUE TITLE
Test 255 [rewind] fails in clean checkout of h3-develop

### ISSUE THREAD

**Issue by npcarter** [EddyRivasLab/hmmer#124]
At least on the Odyssey cluster, test #255 fails when I do a clean checkout of h3-develop, followed by autoconf, configure, make check.  I ran into this while testing the fix to the hmmc2 bug I reported, and decided to try it in a clean checkout because I couldn't see how a change to hmmc2 would affect this test.

**Comment by cryptogenomicon** [comment]
Does not reproduce; works fine for me on ody (Linux) and OS/X, with gcc or icc, with debugging flags or without. Is it reproducible for you, and can you give me more info so I can reproduce? 

To run just that test without having to run the whole testsuite:
 ```
    % cd testsuite
    % ./i21-rewind.pl .. .. tmpfoo
```

The i21-rewind.pl did have an issue that I found (a typo in a commandline path), which I just checked in a fix for, but it was not causing the test to fail.

**Comment by npcarter** [comment]
Hmm,  I can’t replicate this in a clean checkout today, either.  Two possibilities occur to me:
1) Your path fix corrected the issue I was seeing.  Last week, the error I got did look like a path problem, as directly running i21-rewind gave errors about “unable to find hmmsearch in /hmmer/src” or something similar.  (When there was an hmmsearch executable in the src directory of my build).

2) I was doing the wrong thing and building h3-develop against easel-master, which caused the problem.  I ran into this in my first attempt to reproduce the problem today, since h3-develop now expects there to be a file in easel-develop that isn’t present in easel-master.

-Nick
